examples/spring-boot-minimal/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Stage 1 for Build
+FROM openjdk:8-jdk-alpine as buildimage
```

Comment:
Could we use a newer LTS version?

Reply:
thanks for your review. as Alpine is not officially supported, need to change it to:
openjdk:8

But it will not be lighter like Alpine. Is it okay to switch to Ubuntu image?
I will modify accordingly and will do some research on Jib too.

Reply:
@trustin  - I have added LTS image -adoptopenjdk/openjdk8:jdk8u282-b08-alpine

Please check. 

For JIB, will need credentials of target repository where image needs to be published.

Reply:
> For JIB, will need credentials of target repository where image needs to be published.

Yeah, how about letting a user (most likely a dev) specify the credentials via Gradle properties? e.g. `gradle -PdockerUsername=... -PdockerPassword=...`